lua/packer.lua

```diff
@@ -519,6 +550,24 @@ packer.loader_complete = function(lead, _, _)
   return completion_list
 end
 
+-- Snapshot installed plugins
+-- Given a list of plugins it will only snapshot said plugins,
+-- otherwise it will snapshot all plugins (except for disabled)
+packer.snapshot = function (filename)
+  async(function()
+    --local start_time = vim.fn.reltime()
+    log.info(string.format("Taking snapshots of currently installed plugins to %s...", filename))
+    filename = util.join_paths(config.snapshot_path, filename)
+    await(snapshot(filename, plugins))
+    await(a.main)
```

Comment:
We probably don't need to `await` main here with the below profiling line gone.

Reply:
> We probably don't need to `await` main here with the below profiling line gone.

This is the error I get without `await a.main`
```[packer.nvim] [ERROR 15:30:56] async.lua:20: Error in coroutine: ...Downloads/github/packer.nvim.git/snapshot/lua/packer.lua:275: E5560: 
nvim_exec must not be called in a lua loop callback 
```

Line count might be different from the version in the repo since I made some changes